runtime/vm/callin.cpp

```diff
@@ -883,6 +883,24 @@ sendFromMethodDescriptorString(J9VMThread *currentThread, J9UTF8 *descriptor, J9
 				currentThread->returnValue2 = (UDATA)J9VMJAVALANGINVOKEMETHODTYPE_FROMMETHODDESCRIPTORSTRINGAPPENDARG_METHOD(vm);
 			}
 			c_cInterpreter(currentThread);
+			if (VM_VMHelpers::exceptionPending(currentThread)) {
+				j9object_t currentException = currentThread->currentException;
+				J9Class *exceptionClass = J9VMJAVALANGTYPENOTPRESENTEXCEPTION(vm);
+				/* If the current exception is TypeNotPresentException and the cause of TypeNotPresentException is
+				 * ClassNotFoundException, then current exception is set to NoClassDefFoundError. The cause of
+				 * NoClassDefFoundError is set to ClassNotFoundException, and the message of ClassNotFoundException
+				 * is used to set the message of NoClassDefFoundError.
+				 */
+				if (exceptionClass == J9OBJECT_CLAZZ(currentThread, currentException)) {
+					j9object_t cause = J9VMJAVALANGTHROWABLE_CAUSE(currentThread, currentException);
+					J9Class *causeClass = J9VMJAVALANGCLASSNOTFOUNDEXCEPTION(vm);
```

Comment:
Similarly, using name `causeClassThrowable` offers better clarity. In addition, pls define it as `J9Class * const causeClassThrowable`.

Reply:
done - changed to `causeClassCNFE` and added `const`.